pkg/network/tracer/tracer.go

```diff
@@ -451,6 +453,18 @@ func (t *Tracer) getRuntimeCompilationTelemetry() map[string]network.RuntimeComp
 			tm.RuntimeCompilationDuration = duration
 		}
 		result[assetName] = tm
+
+		statsdMetricPrefix := "datadog.system_probe." + assetName
+		tags := []string{fmt.Sprintf("version:%s", version.AgentVersion)}
+
+		if tm.RuntimeCompilationEnabled {
+			if err := statsd.Client.Gauge(statsdMetricPrefix + ".runtime_compilation_result", float64(tm.RuntimeCompilationResult), tags, 1); err != nil {
```

Comment:
Does `Count` make more sense, since we tracking the result of an event?

Reply:
If we use `Count`, the agent will add up all the values received in a time interval. Since we're reporting something which has a continuous value, `Gauge` is a better representation.